reconcile/utils/terraform_client.py

```diff
@@ -683,3 +685,57 @@ def _can_skip_rds_modifications(
             return not set(changed_resource_arguments) - set(changed_values)
         else:
             return False
+
+
+@defer
+def run_terraform(
```

Comment:
I like the idea, but I have some doubts whether it's general enough and/or if we should start with putting it into `terraform_client.py` just yet. Maybe this could move into the integration for now and then we can try to see how reusable it is in the other Cloudflare integrations in a follow-up MR? There will be significant changes to `terraform-cloudflare-resources` as part of supporting the write to Vault feature.

Reply:
In light of significant changes in near future, I've moved this to the integration itself.